### PR TITLE
[EDX-108] Move Control API reference

### PR DESCRIPTION
**Pull request by tbedford**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Move Control API Reference from `/control-api/api-reference` to `/api/control-api`.

* [EDX-54](https://ably.atlassian.net/browse/EDX-54).
* [EDX-108](https://ably.atlassian.net/browse/EDX-108).

## Review

* Check locally.
* Check redirect (from `/control-api/api-reference` to `/api/control-api`).

### CODE REVIEW

File: content/api/control-api.textile [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+title: Control API Reference
 meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically. This is the Control API Reference guide."
 meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
-section: control-api
-index: 10
+section: api
+index: 200
 redoc: true
 spec_url: "https://raw.githubusercontent.com/ably/open-specs/main/definitions/control-v1.yaml"
+redirect_from: /control-api/api-reference
```
Comment (m-hulbert) [inline content/api/control-api.textile:9]: The redirect doesn't work when I test this locally.

**Review by m-hulbert — COMMENTED** [review]
None of the deep links work when I test this locally. I'm not sure if that's because it takes too long for the API reference to load when you run it locally, or if it's a separate issue. 

The main redirect doesn't work either. Did you have any luck with these two running locally @tbedford ?

File: content/api/control-api.textile (hunk 1 above)
Comment (tbedford) [reply content/api/control-api.textile:9]: That's odd. They work fine in the nanoc view: both the deep links and the redirect. However, under the web view neither work (locally). Not sure why not.

Comment (tbedford) [reply content/api/control-api.textile:9]: I've looked into this with Ken's help. There's some website code that needs to be changed to support the new file location. We can flag that up when we do the docs bump containing this change. Ken will make the required change to the website code as part of that docs bump PR.

**Comment by tbedford** [comment]
I fixed some merge conflicts related to updated redirects file.

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**